aspnetcore/toc.yml

```
@@ -1405,7 +1405,7 @@
     - name: 5.0 to 6.0
       displayName: migrate, migration
       uid: migration/50-to-60
-    - name: 5.0 Code to 6.0
+    - name: 3.1,5.0 Code to 6.0
```

```suggestion
    - name: 3.1/5.0 Code to 6.0
```

... or perhaps with a space after the comma as `3.1, 5.0 Code to 6.0`.